docs/userguide.md

```diff
@@ -191,6 +191,22 @@ referenced-dependencies:
   version: 2.1.7
 ```
 
+We also support json-formatted dependencies:
+
```

Comment:
I would consider moving this part down past the license scan dependencies section and adding a `vendored-dependencies` section to the example JSON.

Reply:
I like this suggestion.